main.py

```
@@ -8,18 +9,36 @@
-@bot.message_handler(commands=['start', 'hello'])
-def send_welcome(message):
-    bot.reply_to(message, "Hello! I'm a simple Telegram bot.")
+        Args:
+            message (telebot.types.Message): The message object.
+        """
+        bot.reply_to(message, "Hello! I'm a simple Telegram bot.")
 
-@bot.message_handler(func=lambda msg: True)
-def echo_all(message):
-    bot.reply_to(message, message.text)
+    @bot.message_handler(func=lambda msg: True)
```

Ruff (0.14.14)

[warning] Unused lambda argument: `msg`

(ARG005)